style.css

```diff
@@ -0,0 +1,29 @@
+#img-div {
+    max-width: 100%;
+    text-align: center;
+
+
+}
+#image{
+max-width: 100%;
+margin: 0 auto;
+}
+#tribute-info{
+
+display: flex;
+justify-content: center;
+align-items: center;
+flex-direction: column;
+}
+ul{
+text-align: center;
+margin: 0 auto;
+}
+li{
+text-align: left;
+}
+
+body{
+text-align: center;
+}

```

Comment:
Please consider a class for the `text-align: center;` style since you have used it severally in your css file. This will ensure that you only use it once and maintain clean code.